syz-manager/rpc.go

```diff
@@ -237,6 +237,7 @@ func (serv *RPCServer) NewInput(a *rpctype.NewInputArgs, r *int) error {
 	}
 	serv.corpusCover.Merge(a.Cover)
 	serv.stats.corpusCover.set(len(serv.corpusCover))
+	serv.stats.maxSignal.set(len(serv.maxSignal))
```

Comment:
I think this needs to go into RPCServer.Poll, that's where we update serv.maxSignal.